src/containers/TriggerTemplate/TriggerTemplate.test.js

```diff
@@ -417,16 +417,12 @@ it('TriggerTemplateContainer renders full resource template information', async
   const compareToName = resourceTemplate1Details.metadata.name;
   await waitForElement(() => getByText(compareToName));
   await waitForElement(() => getByText(/revision/i));
-  await waitForElement(() => getByText(/params.gitrevision/i));
```

Comment:
why are these removed ?

Reply:
because they were only rendered in the inactive tab content which is no longer present, they shouldn't have been included in the test in the first place